### PR TITLE
x11-wm/i3-gaps: treeclean

### PR DESCRIPTION
**Pull request by quincyf467**
i3-gaps has been merged into i3wm version 4.22.  Because of this, the i3-gaps upstream repository has been archived by the owner so the x11-wm/i3-gaps package is no longer needed and users will have the additional features that came with i3-gaps in i3wm version 4.22.  No config file changes needed.  Just uninstall x11-wm/i3-gaps and install x11-wm/i3-4.22 and restart your i3 session:)

### CODE REVIEW

File: dev-go/delve/delve-1.20.1.ebuild
Comment (quincyf467) [inline dev-go/delve/delve-1.20.1.ebuild:1]: I'll submit a pull request for dev-go/delve-1.20.2 later.  I didn't want delve to be bundled with my commits to get rid of x11-wm/i3-gaps

**Comment by gksudolol** [comment]
I think last-rite or have i3-gaps a meta/provide package would be more suitable than treeclean

**Comment by ajakk** [comment]
Yes, we generally don't just treeclean packages outside of the normal last-rites process. i3-gaps will be last rited.

**Comment by quincyf467** [comment]
@ajakk got it.  Sorry, still learning the process.  By "last rited" do you mean to create a i3-gaps-9999.ebuild? If so, I'll submit that then

**Comment by gksudolol** [comment]
@quincyf467 See here: https://wiki.gentoo.org/wiki/Project:Treecleaner/Policy

**Comment by quincyf467** [comment]
@gksudolol okay. thank you

**Pull request closed without merging**